crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -1296,7 +1331,8 @@ impl ForeignFrom<(CybersourcePaymentStatus, bool)> for enums::AttemptStatus {
     fn foreign_from((status, capture): (CybersourcePaymentStatus, bool)) -> Self {
         match status {
             CybersourcePaymentStatus::Authorized
-            | CybersourcePaymentStatus::AuthorizedPendingReview => {
+            | CybersourcePaymentStatus::AuthorizedPendingReview
+            | CybersourcePaymentStatus::Accepted => {
```

Comment:
Why are we getting a new status from connector for mandate? Please get it clarified from them

Reply:
Query Raised